### PR TITLE
New package: dcraw

### PR DESCRIPTION
**Pull request by felix**
New package: dcraw

### CODE REVIEW

File: srcpkgs/dcraw/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+version=9.27
+revision=1
+only_for_archs="x86_64 i686"
+build_style=fetch
+makedepends="jasper-devel lcms2-devel"
+short_desc="Convert raw photos"
+maintainer="Felix Hanley <felix@userspace.com.au>"
+license="GPL-2"
+homepage="https://www.cybercom.net/~dcoffin/dcraw/"
+distfiles="https://www.cybercom.net/~dcoffin/dcraw/dcraw.c https://www.cybercom.net/~dcoffin/dcraw/dcraw.1"
```
Comment (chneukirchen) [inline srcpkgs/dcraw/template]: Use newlines here.

File: srcpkgs/dcraw/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'dcraw'
+pkgname=dcraw
+version=9.27
+revision=1
+only_for_archs="x86_64 i686"
```
Comment (chneukirchen) [inline srcpkgs/dcraw/template]: Why?

File: srcpkgs/dcraw/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+makedepends="jasper-devel lcms2-devel"
+short_desc="Convert raw photos"
+maintainer="Felix Hanley <felix@userspace.com.au>"
+license="GPL-2"
+homepage="https://www.cybercom.net/~dcoffin/dcraw/"
+distfiles="https://www.cybercom.net/~dcoffin/dcraw/dcraw.c https://www.cybercom.net/~dcoffin/dcraw/dcraw.1"
+checksum="4a0bc163eca6b896463a56c2b4e45b5b02b039e2654508a5982fe69a01288ec1 f52a6b2ed50360240ff44d7ff9ec430ca8195fbb07f48003a1fa63c5a69217aa"
+
+do_build() {
+	gcc -o dcraw -O4 $CFLAGS dcraw.c $LDFLAGS  -lm -ljasper -ljpeg -llcms2
```
Comment (chneukirchen) [inline srcpkgs/dcraw/template]: -O4 is overwritten by $CFLAGS.

Use $CC.

File: srcpkgs/dcraw/template [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+do_install() {
+	vinstall dcraw 755 /usr/bin
```
Comment (chneukirchen) [inline srcpkgs/dcraw/template]: use vbin

File: srcpkgs/dcraw/template (hunk 2 above)
Comment (felix) [inline srcpkgs/dcraw/template]: Travis didn't like the arm builds and I currently have no way to test them. Should I take it out?

**Comment by felix** [comment]
It seems Travis _does_ like the arm builds!

**Comment by pullmoll** [comment]
The arm build succeeded because by using `$CC` instead of `gcc` you will get `armv7l-linux-gnueabihf-cc` or `armv7l-linux-musleabihf-cc`, the required cross compiler.

**Pull request merged**